### ISSUE TITLE
unzipper parseOne parameters are all optional

### ISSUE THREAD

**Issue by dreamershl** [DefinitelyTyped/DefinitelyTyped#42330]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/unzipper` package and had problems.
  - Authors: @s73obrien

From the code of parseOne, these 2 parameters are optional. I have created the pull request #42329 for this. 
```
function parseOne(match,opts) {
  var inStream = Stream.PassThrough({objectMode:true});
  var outStream = Stream.PassThrough();
  var transform = Stream.Transform({objectMode:true});
  var re = match instanceof RegExp ? match : (match && new RegExp(match));
  var found;

```

**Comment by peterblazejewicz** [comment]
@dreamershl after that's merged, can you close this one? Thanks!

**Comment by dreamershl** [comment]
it is solved by the #42329
